HW3/IO/Write.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.IO;
+
+namespace HW3.IO
+{
+    public static class Write
+    {
+        public static void AppendCredentials(string filename, string username, string password)
+        {
+            CheckForNewLine(filename);
+
+            using (var fs = new FileStream(filename, FileMode.Append))
+            {
+                using (var sw = new StreamWriter(fs))
+                {
+                    sw.WriteLine($"{username}\t{password}");
+                }
+            }
+        }
+
+        private static void CheckForNewLine(string filename)
```

Comment:
Could you please explain the purpose of this method. I don't understand why do you have both reader and writer in the same method.

Reply:
I figured, if the Users.txt is not written by the application but instead comes from another source, I shouldn't assume it ending on a new line and thus appending a new username/password would extend the last password in the list. So that was my rushed approach to make sure I append new credentials in one line each.